src/transaction/recovery.c

```diff
@@ -813,6 +813,12 @@ struct rvfun RV_fun[] = {
    btree_rv_redo_record_modify,
    btree_rv_keyval_dump,
    log_rv_dump_hexa},
+  {RVHF_HEAP_ADD_CHAIN,
```

Comment:
It looks rather confusing. I hope to have a clearer name.
What about `RVHF_APPEND_PAGES_TO_HEAP`?